tests/integration/targets/nxos_user/tests/common/basic.yaml

```diff
@@ -42,21 +44,23 @@
     that:
       - result.changed == true
 
-- name: tearDown
-  register: result
+- name: Attempt to create user with weak password (SHOULD FAIL)
   cisco.nxos.nxos_user:
-    aggregate:
-
-      - name: ansibletest1
+    name: ansibletest_failed
+    configured_password: abc
+  register: result
+  ignore_errors: True
 
-      - name: ansibletest2
+- assert:
+    that: result.failed == True
```

Comment:
Will you be able to validate the error msg here?

Reply:
Added that.